src/backend/cpu/image.hpp

```diff
@@ -11,6 +11,7 @@
 
 #include <Array.hpp>
 #include <graphics_common.hpp>
+#include <af/defines.h>
```

Comment:
You probably don't need this here. defines.h is included from Array.h. You also don't need graphics_common.hpp.